src/vs/workbench/browser/parts/views/viewsService.ts

```diff
@@ -485,7 +485,7 @@ export class ViewsService extends Disposable implements IViewsService {
 	private registerFocusViewAction(viewDescriptor: IViewDescriptor, category?: string | ILocalizedString): IDisposable {
 		return registerAction2(class FocusViewAction extends Action2 {
 			constructor() {
-				const title = localize({ key: 'focus view', comment: ['{0} indicates the name of the view to be focused.'] }, "Focus on {0} View", viewDescriptor.name);
+				const title = localize({ key: 'focus view', comment: ['{0} indicates the name of the view to be focused.'] }, "Focus on {0} View", viewDescriptor.name.value);
 				super({
 					id: viewDescriptor.focusCommand ? viewDescriptor.focusCommand.id : `${viewDescriptor.id}.focus`,
 					title: { original: `Focus on ${viewDescriptor.name} View`, value: title },
```

Comment:
I think you need to change this to `viewDescriptor.name.original` no? This is actually the fix you're fixing. You tested this fix, right?

Reply:
> I think you need to change this to `viewDescriptor.name.original` right? Otherwise you will see `[Object object]` You may want to spot check the other references to make sure you are using the correct value.

Yes, yes, I also found this problem and have already disposed of it. Now it behaves like this in the Chinese environment:

![image](https://github.com/microsoft/vscode/assets/11473889/13dcef28-2108-4138-83e9-5aed021ac090)